### PR TITLE
OSOE-56: Open-source Content Editors

### PR DESCRIPTION
**Pull request by barthamark**
[OSOE-56](https://lombiq.atlassian.net/browse/OSOE-56)

[OSOE-56]: https://lombiq.atlassian.net/browse/OSOE-56?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj [hunk 1]
```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Lombiq.BaseTheme.Tests.UI" Version="3.0.0" />
     <PackageReference Include="Lombiq.ChartJs.Tests.UI" Version="4.0.0" />
+    <PackageReference Include="Lombiq.ContentEditors.Tests.UI" Version="1.0.0-alpha.1.osoe-56" />
```
Comment (Piedone) [inline NuGetTest/test/Lombiq.OSOCE.NuGet.Tests.UI/Lombiq.OSOCE.NuGet.Tests.UI.csproj]: Add a test to the NuGetTest solution (there's a draft in the issue).

File: Readme.md [hunk 2]
```diff
@@ -91,6 +93,7 @@ You can activate various sample content in the site:
   - The "TEST: Basic Orchard Features" setup recipe automatically sets it up. If not using it, run the "Lombiq Orchard Core Base Theme - Layers and Zones" recipe, and then enable the theme in Admin → Design → Themes.
   - In case of theme development you can use the "Lombiq Orchard Core Base Theme - Styling Demo" theme to test against some common HTML elements.
 - [`Lombiq.ChartJs`](https://github.com/Lombiq/Orchard-Chart.js): Go to Recipes in the admin dashboard and select "Lombiq Chart.js - Sample Content - Income/Expense".
+- [`Lombiq.ContentEditors`](https://github.com/Lombiq/Orchard-Content-Editors): Go to Features in the admin dashboard and enable the `Lombiq Content Editors - Sample` feature.
```
Comment (Piedone) [inline Readme.md]: This recipe doesn't exist.

File: Readme.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ Note that this solution also has an Orchard 1 counterpart, [Lombiq's Open-Source
 - Modules
   - [`Lombiq.AuditTrailExtensions`](https://github.com/Lombiq/Audit-Trail-Extensions): A module with additional features for the [Audit Trail module](https://docs.orchardcore.net/en/latest/docs/reference/modules/AuditTrail/) in Orchard Core.
   - [`Lombiq.ChartJs`](https://github.com/Lombiq/Orchard-Chart.js): An Orchard Core wrapper around the [Chart.js](https://www.chartjs.org/) library for displaying datasets as various charts.
+  - [`Lombiq.ContentEditors`](https://github.com/Lombiq/Orchard-Content-Editors): With the help of this module you can create custom multi-page editors with async loading and saving.
```
Comment (Piedone) [inline Readme.md:36]: This is missing: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions#contributing-and-support:~:text=For%20this%2C%20you%27ll%20also%20need%20to%20enable%20the%20feature%20in%20Lombiq.OSOCE.Tests.recipe.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 2 above)
Comment (barthamark) [reply Readme.md]: Yes, it doesn't. I didn't see it necessary actually. But the title here says Samples and Recipes so I thought I add it here. Do you want me to delete it?

Comment (Piedone) [reply Readme.md]: Well, this bullet point references something that doesn't exist. This doesn't make sense. So yes, delete it (if there's no actual need for a recipe).

Comment (Piedone) [reply Readme.md]: Still.

**Review by Piedone — CHANGES_REQUESTED** [review]

**Review by Piedone — APPROVED** [review]

**Review by Piedone — APPROVED** [review]

**Comment by Piedone** [comment]
The failing Windows build is expected, see https://lombiq.atlassian.net/browse/OSOE-620.

**Pull request merged**